qiskit_experiments/calibration/__init__.py

```diff
@@ -0,0 +1,72 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+r"""
+Qiskit Experiments Calibration Root.
+
+Calibrations are managed by the Calibrations class. This class stores schedules which are
+intended to be fully parameterized, including the index of the channels. This class:
+- supports having different schedules share parameters
+- allows default schedules for qubits that can be overridden for specific qubits.
+
+Parametric channel naming convention.
+Parametrized channel indices must be named according to a predefined pattern to properly
+identify the channels and control channels when assigning values to the parametric
+channel indices. A channel must have a name that starts with `ch` followed by an integer.
+For control channels this integer can be followed by a sequence `.integer`.
+Optionally, the name can end with `$integer` to specify the index of a control channel
```

Comment:
In a cross-resonance architecture you never need more than one control channel per qubit pair. There are however different architectures that need more than one control channel per qubit pair. A good example of this is a tunable coupler with an iSWAP and a bSWAP gate. E.g. https://arxiv.org/pdf/1708.02090.pdf

In setups where the same qubit pair has several control channels these control channels will have different purposes so the schedules played on them will be different. If this is not that case then it does not make sense to have more than one control channel per qubit pair. Therefore we will not need to define two entries with just different parameter name as you suggest.

Reply:
> these control channels will have different purposes so the schedules played on them will be different

If this is always true, perhaps this would be fine. But how we guarantee the ordering of channel? Perhaps this is the issue of backend configuration implementation. This should be able to provide control channel with purpose (not only with Hamiltonian).